shared/js/intl/l20n.js

```diff
@@ -1837,8 +1837,12 @@
 
       const pseudoentries = Object.create(null);
       for (let key in entries) {
-        pseudoentries[key] = walkEntry(
-          entries[key], pseudo[lang.code].process);
+        if (key in ['datePickerOrder', 'timePickerOrder', 'dueationPattern']) {
```

Comment:
`dueationPattern` has a typo, but more importantly, I don't think that's how the `in` operator works, does it?